### PR TITLE
[New] Clear cache guide

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- Created a guide to clear cache for chrome, safari, firefox, and edge.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 50d28f3c9f2b736c29d6a966207488204c40bea3

https://deploy-preview-3182--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by Rajakavitha1** [comment]
Here's the preview link:https://deploy-preview-3182--nostalgic-ptolemy-b01ab8.netlify.com/quick-answers/websites/clear-cache-shortguide/

File: docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+When you clear the cache the performance of the system and application is enhanced.
+
+## Clear the Cache in Chrome
+
+### Keyboard Shortcut
+
+> **On Windows or Linux**
+>
+> While in the browser press **Ctrl** + **shift** + **Delete** keys.
```
Comment (andystevensname) [inline docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md]: Is there a reason we're not capitalizing shift here?

File: docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+## Clear the Cache in Safari
+
+1. From the **Safari** menu, select **Clear History**.
+   ![Safari Clear History](cache-safari.png)
+2. Select the desired time range option in the **Clear** drop-down menu.
+
+      To clear all the cache select **all history**, and then click **Clear History**.
+
+3. Go to the **Safari** menu and click **Quit Safari** to exit the browser.
```
Comment (andystevensname) [inline docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md]: Why are we exiting out of Safari?

File: docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+> **On Windows or Linux**
+>
+> While in the browser press **Ctrl** + **shift** + **Delete** keys.
+
+> **On MacOS**
+>
+> While in the browser press **Command** + **shift** + **Delete** keys.
+
+![Firefox Clear Recent History](cache-firefox.png "Firefox Clear Recent History")
```
Comment (andystevensname) [inline docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md]: Should this be the start of the "In the Browser" section?

File: docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+- Additionally, tabs in a private browser window are isolated from each other and private browsing windows can not be passed to other Apple devices via Handoff.
+- Data is cleared upon exiting the browser window.
+
+1.  To open a private browsing session, click the **File** menu and select **New Private Window**.
+
+      - You can also open a private window with the keyboard shortcut **Ctrl** + **Shift** + **N** (Windows and Linux) or **Command** + **Shift** + **N** (MacOS).
+
+## Clear the Cache in Edge
+
+1. While in the browser press **Ctrl** + **Shift** + **Delete** keys. The **Clear browsing data** window appears.
```
Comment (andystevensname) [inline docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md]: We should make this section look like the other keyboard shortcut sections, with a blockquote, just to be consistent.

File: docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+
+      If you are using MacOS, press **Command** + **Shift** + **Delete** keys.
```
Comment (andystevensname) [inline docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md]: ...then, we should make an H2 here for "In the Browser"

**Comment by Rajakavitha1** [comment]
hi @andystevensname  incorporated the changes that you suggested. Introduced a new heading In the Browser section and ensured that the capitalization and key board short cuts are consistent. Please let me know if any other changes need to be made.

**Pull request merged**